### DISCUSSION TITLE
Faltou incluir algo para criptografar password do banco.

### DISCUSSION THREAD

**Discussion by Lcrmartins** [balta-io/7194#16] [Perguntas e Respostas] [answered]
Prezado Balta.
Muito bons cursos.
Sou novato, mas já aprendi que é importante não salvar senhas (password) nos bancos. Como até aqui nada foi mencionado, teremos aula sobre isso mais adiante? Estou na carreira Backend. Obrigado.
Luis Martins

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @Lcrmartins, você pode usar o pacote [Secure Identity](https://github.com/andrebaltieri/SecureIdentity) que isola o hasher do ASP.NET Identity... ou usar o ASP.NET Identity que já fará o Hash da senha para você!
